src/fhirtypes/ElementDefinition.ts

```diff
@@ -1924,7 +1924,7 @@ export class ElementDefinition {
    * @throws {ValueAlreadyAssignedError} when the value is already assigned to a different value
    * @throws {TypeNotFoundError} when the value does not match the type of the ElementDefinition
    */
-  private assignString(value: string, exactly = false): void {
+  private async assignString(value: string, exactly = false): Promise<void> {
```

Comment:
Does this really work?  Wouldn't you need to change the code that calls `assignString` so that it awaits the response now?  And then that would make that code `async`, which means anything that calls it needs to `await` too... and on and on.  In other words, I'm not sure that we can easily change this to `async` without causing a cascade of other changes.

According to [this issue](https://github.com/terser/html-minifier-terser/issues/78), the `minify` function was changed to async in v6.x.  Would using the most recent 5.x version (5.1.1) work for you?  Then `minify` could stay synchronous.

Reply:
I moved html-minifier-terser to dependencies and downgraded to the synchronous version to avoid the overhaul issues you had mentioned.  The FSH conversion still works from my end of things.
Let me know if you need any other updates. Thanks!